src/components/mixins/DownloadMixin.vue

```diff
@@ -46,6 +52,35 @@ export default class DownloadMixin extends Vue {
         }
     }
 
+    downloadProgressCallback(assignId: number, progress: number, modName: string, status: number, err: R2Error | null) {
```

Comment:
These now have some repetition I think is addressed in #1625

Reply:
correct 👍